File: src/ExamRegister.WebApi.Abstractions/DTO/GrupodeMedicoDTO.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ExamRegister.WebApi.Abstractions.DTO {
    public partial class GrupodeMedicoDTO {
        public Guid idgrupodemedicos { get; set; }
        public string idexterno { get; set; }
        public string nome { get; set; }
        public DateTime? inativo { get; set; }

        public MedicoGrupoDTO[] MedicoGrupo { get; set; }
    }
}

File: src/ExamRegister.UI/src/app/shared/DTOs/grupodemedico-dto.ts
import { MedicogrupoDTO } from "./medicogrupo-dto";

export interface GrupodemedicoDTO {
    idgrupodemedicos: string;
    idexterno: string;
    nome: string;
    inativo?: Date;

    medicoGrupo: MedicogrupoDTO[];
}
